### PR TITLE
Adds example on usage sphere obstacles and urdf obstacles in environments.

### PR DESCRIPTION
**Pull request by maxspahn**
Making use of motionPlanningScenes, this implements an easy way to integrate obstacles into the environments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ In this package, generic urdf robots and a panda gym environment are available.
 
 This package depends on casadi for dynamics generation and gym.
 Dependencies should be installed through pip installation, see below.
+When using obstacles and goals, this package also depends on MotionPlanningScenes,
+see https://github.com/maxspahn/motion_planning_scenes
+
+You can either install that manually as explained in the corresponding readme or 
+you install it through
+```bash
+pip3 install -r requirements_scenes.py
```
Comment (GijsGroote) [inline README.md:15]: This should be requirements_scenes.txt not .py

**Comment by GijsGroote** [comment]
It feels really clean to be able to just do:
``` env.addObstacle(sphereObst1) ```
I think a new folder (inside urdfGymExamples) should be created to keep example support files like obstacles.py, goal.py and obstacleData for easy access.

File: README.md (hunk 1 above)
Comment (maxspahn) [reply README.md:15]: Thanks. Anything else in the branch?

Comment (GijsGroote) [reply README.md]: I could not find any other points to criticize, thanks for adding this functionality. I will make quite some use of it :)

**Pull request merged**